### PR TITLE
Enhancement: Trello Board objects - Added resolver function for populating "Get Board" field

### PR DESCRIPTION
**Pull request by mcat-ee**
Hi all,

There's been a standing frustration with the Trello node, specifically that it doesn't populate the Board field, which (previously) meant you needed to go to your board in a web browser, add .json onto the URL while viewing a ticket (but not just the board itself as this will cause an error!), search through the JSON representation of your board, and paste it back into Trello.

That should no longer be the case.

I've written a resolve function based on the Google Calendar loading function in _that_ node.

Can I get some feedback on how this code reads? Please do not merge it yet, as I'd like to run some more tests (but again, I'd really like some feedback as I rarely use Typescript).

Additionally I'd like to flag some future work; it would be good to manage the Trello API requests a bit better, they seem to be performed in a way inconsistent with other, newer nodes (If anyone is curious I'm happy to tag examples in a comment to this PR)

### CODE REVIEW

**Comment by mcat-ee** [comment]
@RicardoE105 it is!

**Comment by Susangriffis** [comment]
Hello 


Sent from Yahoo Mail for iPhone


On Sunday, February 27, 2022, 19:06, Tom McAtee ***@***.***> wrote:




@RicardoE105 it is!

File: packages/nodes-base/nodes/Trello/Trello.node.ts [hunk 1]
```diff
@@ -135,6 +139,30 @@ export class Trello implements INodeType {
...
+						const boardId = board.id;
+						returnData.push({
+							name: boardName,
+							value: boardId,
+						});
+					}
+					return returnData;
+				}
+		}
+	};
```
[the remark this answers is not in the document]
Comment (michael-radency) [reply packages/nodes-base/nodes/Trello/Trello.node.ts]: please do not try to merge directly to master use `git push upstream  enhancement/trello-get-all-boards-for-user`

[the remark this answers is not in the document]
Comment (michael-radency) [reply packages/nodes-base/nodes/Trello/Trello.node.ts]: please let me know when you finish tests

**Comment by michael-radency** [comment]
Hello @mcat-ee made some updates to get all boards operation to be more consistent with other nodes, you can now get board ID by using Get All operation

**Comment by RicardoE105** [comment]
@mcat-ee, in order to merge this PR, the contributor license agreement needs to be signed. It looks like you missed signing it with the `Tom` user. Please let us know when is signed as it's the only thing missing. Thanks

**Comment by nivb06** [comment]
Hi @mcat-ee 
I would have to close this PR as you have not signed the contributor licence agreement (CLA) agreement. 
Feel free to reopen it once it is signed

**Pull request closed without merging**